### PR TITLE
sql: update parameters and examples of load data

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update parameters and examples of `load data`

close https://github.com/pingcap/docs/issues/2719

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2568, https://github.com/pingcap/docs-cn/pull/3335
- Other reference link(s):

### CODE REVIEW

**Comment by ran-huang** [comment]
https://github.com/pingcap/docs-cn/pull/2568 applies to master, v4.0, v3.1, and v3.0.
https://github.com/pingcap/docs-cn/pull/3335 applies to master and v4.0.
I'll edit the cherry-picked PRs accordingly.

File: sql-statements/sql-statement-load-data.md [hunk 1]
```diff
@@ -15,10 +15,52 @@ The `LOAD DATA` statement batch loads data into a TiDB table.
...
+If you do not specify the processing parameters, the imported data is processed in the following manner:
+
+```sql
+FIELDS TERMINATED BY '\t' ENCLOSED BY ''
+LINES TERMINATED BY '\n'
+```
+
+### `IGNORE number LINES`
+
+You can ignore the first `number` lines of the file by configuring the `IGNORE number LINES` parameter. For example, if you configure `IGNORE 1 LINES`, the first line of the file is ignored.
```
Comment (TomShawn) [inline sql-statements/sql-statement-load-data.md:54]:
```suggestion
You can ignore the first `number` lines of a file by configuring the `IGNORE number LINES` parameter. For example, if you configure `IGNORE 1 LINES`, the first line of a file is ignored.
```

File: sql-statements/sql-statement-load-data.md [hunk 2]
```diff
@@ -15,10 +15,52 @@ The `LOAD DATA` statement batch loads data into a TiDB table.
...
+
+In addition, for the `DuplicateOpt`, `CharsetOpt`, and `LoadDataSetSpecOpt` parameters, TiDB currently only supports parsing syntax.
```
Comment (TomShawn) [inline sql-statements/sql-statement-load-data.md:56]:
```suggestion
In addition, TiDB currently only supports parsing the syntax of the `DuplicateOpt`, `CharsetOpt`, and `LoadDataSetSpecOpt` parameters.
```

File: sql-statements/sql-statement-load-data.md [hunk 3]
```diff
@@ -30,10 +72,23 @@ mysql> CREATE TABLE trips (
...
     -> );
+```
+
+```
 Query OK, 0 rows affected (0.14 sec)
+```
+
+The following example imports data using `LOAD DATA`. Comma is specified as the separating character. The double quotation marks that enclose the data is ignored. The first line of the file is ignored.
+
+If you see the error message `ERROR 1148 (42000): the used command is not allowed with this TiDB version`, refer to [ERROR 1148 (42000): the used command is not allowed with this TiDB version](/faq/tidb-faq.md#error-1148-42000-the-used-command-is-not-allowed-with-this-tidb-version)
```
Comment (TomShawn) [inline sql-statements/sql-statement-load-data.md:83]:
```suggestion
If you see the error message `ERROR 1148 (42000): the used command is not allowed with this TiDB version`, refer to [ERROR 1148 (42000): the used command is not allowed with this TiDB version](/faq/tidb-faq.md#error-1148-42000-the-used-command-is-not-allowed-with-this-tidb-version).
```

File: sql-statements/sql-statement-load-data.md [hunk 4]
```diff
@@ -15,10 +15,52 @@ The `LOAD DATA` statement batch loads data into a TiDB table.
...
+
+You can specify that the imported data file is located on the client or server by configuring the `LocalOpt` parameter. Currently TiDB only supports data import from the client, so when importing data, set `LocalOpt` to `Local`.
+
+### `Fields` and `Lines`
+
+You can specify how to process the data format by configuring the `Fields` and `Lines` parameters.
+
+- `FIELDS TERMINATED BY`: Specify the separating character of the data.
+- `FIELDS ENCLOSED BY`: Specify the enclosing character of the data.
+- `LINES TERMINATED BY`: Specify the line terminator, if you want to end a line with a certain character.
```
Comment (TomShawn) [inline sql-statements/sql-statement-load-data.md:30]:
```suggestion
- `LINES TERMINATED BY`: Specifies the line terminator, if you want to end a line with a certain character.
```

**Comment by yikeke** [comment]
PTAL @TomShawn @imtbkcat

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**